services/users.js

```diff
@@ -33,9 +33,8 @@ async function addFriend (userId, friendFName, friendLName, friendEmail) {
     friendId = await UserModel.getUserIdByEmail(friendEmail)
   } else {
     const secret = uuidv4()
-    const { friendId } = await UserModel.create(friendFName, friendLName, friendEmail, secret)
+    friendId = await UserModel.create(friendFName, friendLName, friendEmail, secret)
     // TODO: send invitation email
-    return friendId
```

Comment:
Need to rebase this on master. That is, master has a new commit, so we want to chop off our code from this branch, and replay it on top of master (including the new commit). 

To rebase, follow the below instructions. (or lookup a guide lmao). If you fuck up and/or want to restart at any time, just `git reset --hard`, which will reset your code and your commit logs back to what you last pushed to github. 

1. `git branch -b backup` This copies your current branch into master (so if you fuck up everything, you can always delete the branch and rename this one, just like new).
1. `git checkout master`
1. `git pull` This makes sure master is up to date.
1. `git checkout  <this-branch-name>`
1. `git log`. Take note on what the logs look like (you should see your 5 or so commits, plus the previous master commit)
1. `git pull --rebase origin master`. This initiates the rebase on top of master. If there's no merge conflict, skip to the end.
1.  It'll tell you there's a merge conflict. Follow the instructions to fix it (read carefully, google if you're not sure). The last step should be to `git rebase --continue`.
1. CHECK: If you `git status`, it should tell you that your branch is 1 (or maybe 6?) commits ahead of master. If you `git log`, you should see the new master commit placed before the rest of your branch commits. 
1. `git push -f` Force pushing is required to override what you've pushed to github.

Reply:
Ahh yes thanks for the step by step. I was starting to worry about versions getting mixed up. This really helps to clarify things.